Recast/iTunesPodcasts/Models/RSS/Podcast.m

```diff
@@ -0,0 +1,81 @@
+//
+//  Podcast.m
+//  Recast
+//
+//  Created by Mindy Lou on 10/21/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "Podcast.h"
+
+@implementation Podcast
+
+@dynamic collectionId;
+@dynamic feedUrl;
+@dynamic artistName;
+@dynamic collectionName;
+@dynamic artworkUrl30;
+@dynamic artworkUrl60;
+@dynamic artworkUrl100;
+@dynamic artworkUrl600;
+@dynamic collectionExplicitness;
+@dynamic primaryGenreName;
+@dynamic genreIds;
+@dynamic genres;
+@dynamic title;
+@dynamic link;
+@dynamic descriptionText;
+@dynamic language;
+@dynamic copyright;
+@dynamic managingEditor;
+@dynamic webMaster;
+@dynamic pubDate;
+@dynamic lastBuildDate;
+@dynamic categories;
+@dynamic generator;
+@dynamic docs;
+@dynamic cloud;
+@dynamic rating;
+@dynamic ttl;
+@dynamic image;
+@dynamic items;
+@dynamic iTunes;
+@dynamic rawSkipDays;
+@dynamic textInput;
+@dynamic skipHours;
+
+- (void)combineWithPodcast:(id<PartialPodcast>)podcast {
```

Comment:
how do we call this function in swift?

Reply:
bc of the swift bridging header it's still `podcast.combine(with: partial)` #swifty